app/src/components/results/noResultsFound.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  Flex,
+  Heading,
+  Text,
+  Link,
+  List,
+} from "@nypl/design-system-react-components";
+import Image from "next/image";
+import useBreakpoints from "@/src/hooks/useBreakpoints";
+import { useFeedbackContext } from "@/src/context/FeedbackProvider";
+
+export default function NoResultsFound({ searchTerm }) {
+  console.log("searchTerm is: ", searchTerm);
```

Comment:
Remove the console eventually and after you can better log this error. This is a server component, so you should be able to use the logger (for cloudwatch), right?